worker-workflow-testing/src/main/java/com/github/cafdataprocessing/workflow/testing/models/ApplicationMock.java

```diff
@@ -67,4 +67,16 @@ public ConfigurationSource getConfigurationSource()
         return configurationSource;
     }
 
+    @Override
+    public String getName()
+    {
+        return "application-worker-base";
+    }
+
+    @Override
+    public String getVersion()
+    {
+        return "1.0.0-SNAPSHOT-APPLICATION";
+    }
+
```

Comment:
Why isn't this just `worker-base` and `1.0.0-SNAPSHOT` so that you don't have to make all those `WorkflowControlTest.java` changes?

Reply:
I wanted to make sure that the values are coming from the Application and not for the DocumentWorkerConfiguration. The other way was to change the value set in the DocumentWorkerConfiguration during createDocument method call in all test cases. Was not sure the usage of createDocument in other places.. Just updated the affected testcases.